tsdb/index/postings.go

```diff
@@ -79,6 +79,56 @@ func (p *MemPostings) SortedKeys() []labels.Label {
 	return keys
 }
 
+type PostingsStats struct {
+	CardinalityMetricsStats []Stat
+	CardinalityLabelStats   []Stat
+	LabelValueStats         []Stat
+	LabelValuePairsStats    []Stat
+}
+
+// Calculating Cardinality Stats
```

Comment:
Format of this comment should be something like `// Stats calculates the cardinality statistics for postings.`